.github/workflows/tests.yml

```diff
@@ -48,7 +48,7 @@ jobs:
   forest-sync-check:
     if: ${{ contains(github.event.pull_request.labels.*.name, 'Release') }}
     name: forest calibnet sync check
-    runs-on: buildjet-8vcpu-ubuntu-2004
+    runs-on: fuzzy
```

Comment:
We should disable `SCCACHE_GHA_ENABLED` on `fuzzy` and use local FS as cache backend maybe